### PR TITLE
Dev/nat theotokos antiphon1

### PR DESCRIPTION
**Pull request by JedBartlett**
I've added the antiphon, and I think I got all your e-mail contents captured here.

### CODE REVIEW

**Comment by JedBartlett** [comment]
[NativityTheotokos-Antiphon1-rdrG.pdf](https://github.com/JedBartlett/OrthodoxLiturgics/files/2351456/NativityTheotokos-Antiphon1-rdrG.pdf)

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  \set Timing.defaultBarType = "invisible" %% Only put bar lines where I say
+}
+
+refrain = \lyricmode {
+  Through the in -- ter -- ces -- sions of the The -- o -- to -- kos, __ O __ Sav -- ior, save __ us.
+}
+verseOne = \lyricmode {
+  % The verses can be broken up further than they are in the Liturgy variable text
+  % http://ww1.antiochian.org/sites/default/files/nativity_of_theotokos_series.pdf
+  Re -- mem -- ber, O Lord, Dav -- id and all his meek -- ness.
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Da -- vid

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+  \refrain
+  Lo, we have heard it at Eph -- ra -- tha.
+    We have found it in the plains of the wood.
+  \refrain
+  Glo-- ri -- ous things are spo -- ken of thee,
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Space after Glo

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    O ci -- ty of God.
+  \refrain
+  God is in the midst of her; she shall not be sha -- ken;
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: shak -- en

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+    God shall help her right ear -- ly in the morn -- ing.
+  \refrain
+  Glo -- ry to the \recite "Father and to the Son and to" the Ho -- ly Spi -- rit,
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Spir -- it

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+  g4 g g g\breve g4 fis2 fis4 fis d2 r4 \bar "||"
+  \altref \bar "|." % Refrain
+}
+
+tenrefmost = { b4 c d d d d d d c d e2 d4( e) d( c) b2 c2 d1( c) b1 }
+tenref = { \tenrefmost \bar "||" }
+tenor = \relative c' {
+  \global
+  % Verse 1
+  b4 c d d d e2 e4 d4 r4 \bar "|"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: On "David" e2 e4 should be changed to e2. for a dotted half note

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+sopref = { \soprefmost \bar "||" }
+soprano = \relative g' {
+  \global
+  % Verse 1
+  g4 g g a b c2. b4 r4 \bar "|"
+  a4 b g a2. g2 \bar "||"
+  \sopref % Refrain
+  % Verse 2
+  g4 g a b a b c2. c4 b2 r4 \bar "|"
+  g4 a b b b g a2 a4 a g2. r4 \bar "||"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Remove rest for consistency with other phrases

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+altref = { \altrefmost \bar "||" }
+alto = \relative g' {
+  \global
+  % Verse 1
+  g4 g g g g g2. g4 r4 \bar "|"
+  g4 g g fis2. d2 \bar "||"
+  \altref % Refrain
+  % Verse 2
+  g4 g g g g g g2. g4 g2 r4 \bar "|"
+  g4 g g g g g g2 g4 g d2. r4 \bar "||"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Should move to F# on "plains", no rest at the end to stay consistent with the other phrases.
g4 g g g g g fis2 fis4 fis d2. \bar "||"

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+tenref = { \tenrefmost \bar "||" }
+tenor = \relative c' {
+  \global
+  % Verse 1
+  b4 c d d d e2 e4 d4 r4 \bar "|"
+  d4 d b c2. b2 \bar "||"
+  \tenref % Refrain
+  % Verse 2
+  b4 b c d c d e2. e4 d2 r4 \bar "|"
+  b4 c d d d b c2 c4 c b2. r4 \bar "||"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Remove rest for consistency with other phrases

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+basref = { \basrefmost \bar "||" }
+bass = \relative c {
+  \global
+  % Verse 1
+  g'4 g g g g c,2 e4 g4 r4 \bar "|"
+  g4 g g d2. g2 \bar "||"
+  \basref % Refrain
+  % Verse 2
+  g4 g g g g g c,2. e4 g2 r4 \bar "|"
+  g4 g g g g g c,2 c4 e g2. r4 \bar "||"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Should stay on a D through "plains of the", no rest for consistency.
g4 g g g g g d2 d4 d g2. \bar "||"

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+  % Verse 1
+  b4 c d d d e2 e4 d4 r4 \bar "|"
+  d4 d b c2. b2 \bar "||"
+  \tenref % Refrain
+  % Verse 2
+  b4 b c d c d e2. e4 d2 r4 \bar "|"
+  b4 c d d d b c2 c4 c b2. r4 \bar "||"
+  \tenref % Refrain
+  % Verse 3
+  b4 b c d d e2. c4 e d2 r4 \bar "|"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: On "spoken of", moving from C to E is good for maintaining interest in the phrase, but unfortunately it causes parallel octaves with the bass, which makes it sound weird. Instead you should either stay on E or, if you want to keep an interesting phrase in the tenor here, you can make it a voice exchange with the bass by swapping the notes around, repeating the E then moving to the C and finishing on D.

So basically change the c4 e to e4 e or e4 c

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 11]
```diff
@@ -0,0 +1,189 @@
...
+  b4 c2 c4 c b2. \bar "||"
+  \tenref % Refrain
+  % Verse 4
+  b4 c d d d d d d d b c e2. d4 r4 \bar "|"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: On "not be", the B and C sound a bit dissonant, you probably want C and D instead. So change b c e2. to c d e2.

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 12]
```diff
@@ -0,0 +1,189 @@
...
+  % Verse 2
+  g4 g g g g g g2. g4 g2 r4 \bar "|"
+  g4 g g g g g g2 g4 g d2. r4 \bar "||"
+  \altref % Refrain
+  % Verse 3
+  g4 g g g g g2. g4 g g2 r4 \bar "|"
+  g4 fis2 fis4 fis d2. \bar "||"
+  \altref% Refrain
+  % Verse 4
+  g4 g g g g g g g g g g g2. d4 r4 \bar "|"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: I would keep this phrase similar to the others by staying on the G. That way it stands out more in the refrain when the alto skips the F# straight down to D. So change g2. d4 to g2. g4

**Review by ghost — CHANGES_REQUESTED** [review]
Has a few errors but it's almost there.

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly (hunk 10 above)
Comment (JedBartlett) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: I like the second option here, otherwise the tenor isn't adding anything to this phrase.

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly (hunk 11 above)
Comment (JedBartlett) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Yea, that's what I meant, doesn't sound right at all when I play it.

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly (hunk 7 above)
Comment (jbhak4) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Rest was removed, still need to change the last 3 Gs to F#s

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly [hunk 13]
```diff
@@ -0,0 +1,189 @@
...
+altref = { \altrefmost \bar "||" }
+alto = \relative g' {
+  \global
+  % Verse 1
+  g4 g g g g g2. g4 r4 \bar "|"
+  g4 g g fis2. d2 \bar "||"
+  \altref % Refrain
+  % Verse 2
+  g4 g g g g g g2. g4 g2 r4 \bar "|"
+  g4 g g g g g g2 g4 g d2. \bar "||"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Still need to change last 3 Gs to F#s, like this:
g4 g g g g g fis2 fis4 fis d2. \bar "||"

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly (hunk 10 above)
Comment (ghost) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon1-rdrG.ly]: Yeah me too

**Review by ghost — APPROVED** [review]
